frontend/store/reducers/userReducer.js

```diff
@@ -72,8 +68,6 @@ export const userLogin = async (response, dispatch) => {
     if (user && user.token) {
       dispatch({
         type: UserTypes.RECEIVED_LOGIN,
-        payload: user,
```

Comment:
@jmawloo , did you intend to remove this line ?

Reply:
hey steven, just had a chance to see this. I tried testing on my end and nothing bad happened when I removed the lines but if testing becomes too tedious you can discard the changes :). The original intention was to clean up code that I believe was unused.